pkg/session/dummy_holder.go

```diff
@@ -12,15 +12,20 @@ type DummySessionParamHandler struct {
 }
 
 // BindParamFormatCodes implements SessionParamsHolder.
```

Comment:
Here `implements SessionParamsHolder` can be kept, only add it to DistributionIsDefault function

Reply:
fixed